public/form/login.html

```diff
@@ -2,34 +2,64 @@
 
```

Comment:
Can you modify the `<html>` element to be `<html lang="en">`. It's just a small step toward accessibility ;).